tests/ng2web/src/app/account.ngrx.spec.ts

```diff
@@ -0,0 +1,61 @@
+/* tslint:disable:no-unused-variable */
+
+import { TestBed, async, inject } from '@angular/core/testing';
+import { StoreModule, Store } from '@ngrx/store';
+import { EffectsModule } from '@ngrx/effects';
+import { RouterStoreModule } from '@ngrx/router-store';
+import { AppComponent } from './app.component';
+import { AppEffects } from './shared/app.effects';
+import * as fromApp from './shared/app.reducer';
+import { SDKBrowserModule, LoopbackEffects, LoopbackReducer, LoopbackStateInterface } from './shared/sdk/index';
+import { User, AccessToken } from './shared/sdk/models';
+import { UserApi } from './shared/sdk/services';
+import { AccountActions, getLoopbackAuthState, SDKToken } from './shared/sdk';
+
+const Helpers: {
+  create: Function
+} = {
+  create : function() {
+    const user: User = new User();
+    user.email = Date.now() + '@test.com';
+    user.password = 'test';
+    return user;
+  }
+};
+
+describe('Ngrx: Account', () => {
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        SDKBrowserModule.forRoot(),
+        StoreModule.provideStore(Object.assign({}, LoopbackReducer, {
```

Comment:
What about using the SDKBrowserModule Class to provide a method to return whatever is provided by the SDK, this way is easier to learn and understand, even taking advantage of intellisense.... Something like:

````js
imports: [
         SDKBrowserModule.forRoot(),
         StoreModule.provideStore(Object.assign({}, SDKBrowserModule.getReducer(), {
            app: SDKBrowserModule.getFromApp().reducer
         }))
]
````

This way is: 

1- Easier to explore.
2- Cleaner since you avoid a bunch of imports.